lib/TestScriptRunnable.rb

```diff
@@ -80,25 +100,16 @@ def get_resource_from_ref reference
     end
   end
 
-  def action_delete(sourceId, type)
-    FHIR::TestScript::Setup::Action.new({
-      operation: FHIR::TestScript::Setup::Action::Operation.new({
-        sourceId: sourceId,
-        resource: type,
-        local_method: 'delete'
-      })
-    })
-  end 
-
-  def action_create(sourceId, type)
-    FHIR::TestScript::Setup::Action.new({
-      operation: FHIR::TestScript::Setup::Action::Operation.new({
-        sourceId: sourceId,
-        resource: type,
-        local_method: 'delete'
-      })
-    })
-  end 
+  # Failure of postprocessing (autodelete) won't fail test as opposed to failure of autocreate will.
+  def postprocess 
+    script.fixture.each_with_object({}) do |fixture|
+      if fixture.autodelete
+        FHIR.logger.info "[.load_fixture] Autodelete Fixture: #{fixture.id}"
+        resource = get_resource_from_ref(fixture.resource)
+        reply = client.destroy(resource.class, resource.id)
+      end
+    end
```

Comment:
```suggestion
    fixtures.each do |fixture|
      if fixture.autodelete
        FHIR.logger.info "[.load_fixture] Autodelete Fixture: #{fixture.id}"
        client.destroy(fixture.resourceType, id_map[fixture.id])
      end
    end
```
The `fixtures` instance variable stores each fixture as a `FHIR::Resource` (i.e. something like `fixture-patient-create` gets stored as a `FHIR::Patient`). So, by iterating through `fixtures`, we don't need to call `get_resource_from_ref ` since the objects stored within `fixtures` are already `resources` and not references.

Here's some documentation on the [`each`](https://apidock.com/ruby/Array/each) method and the [`each_with_object`](https://apidock.com/rails/Enumerable/each_with_object) method. If you're unsure why we would just use `each` in this situation, let me know. 

Last, we want to give `client.destroy` the resource's **_server_** id, not its static id -- this is where/why `id_map` gets used. So, our `id_map` holds our mapping from static id ==> server id.

Reply:
All make sense